### ISSUE TITLE
Unhandled exception (Application.Current.DispatcherUnhandledException)

### ISSUE THREAD

**Issue by ImJustMango** [RatScanner/RatScanner#175]
**Error**
Unhandled exception (Application.Current.DispatcherUnhandledException)
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.PageSwitcher.OnClosed(EventArgs e)
   at System.Windows.Window.InternalClose(Boolean shutdown, Boolean ignoreCancel)
   at System.Windows.Application.DoShutdown()
   at System.Windows.Application.ShutdownImpl()
   at System.Windows.Application.ShutdownCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```
<details>
<summary>Log</summary>

```
[02:25:09.3891997] > [Warning] Config version (-1) is not supported!
[02:25:09.4037518] > [Info]  Detected 5120x1440 Resolution.

You can adjust this inside the settings.
[02:25:09.4195690] > [Error] Unhandled exception (Application.Current.DispatcherUnhandledException)
 -------------------- 
 System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.PageSwitcher.OnClosed(EventArgs e)
   at System.Windows.Window.InternalClose(Boolean shutdown, Boolean ignoreCancel)
   at System.Windows.Application.DoShutdown()
   at System.Windows.Application.ShutdownImpl()
   at System.Windows.Application.ShutdownCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
[02:25:26.7896904] > [Warning] Config version (-1) is not supported!
[02:25:26.8043404] > [Info]  Detected 5120x1440 Resolution.

You can adjust this inside the settings.
[02:25:26.8208802] > [Error] Unhandled exception (Application.Current.DispatcherUnhandledException)
 -------------------- 
 System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.PageSwitcher.OnClosed(EventArgs e)
   at System.Windows.Window.InternalClose(Boolean shutdown, Boolean ignoreCancel)
   at System.Windows.Application.DoShutdown()
   at System.Windows.Application.ShutdownImpl()
   at System.Windows.Application.ShutdownCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
[02:26:18.5131842] > [Warning] Config version (-1) is not supported!
[02:26:18.5275052] > [Info]  Detected 5120x1440 Resolution.

You can adjust this inside the settings.
[02:26:18.5413776] > [Error] Unhandled exception (Application.Current.DispatcherUnhandledException)
 -------------------- 
 System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.PageSwitcher.OnClosed(EventArgs e)
   at System.Windows.Window.InternalClose(Boolean shutdown, Boolean ignoreCancel)
   at System.Windows.Application.DoShutdown()
   at System.Windows.Application.ShutdownImpl()
   at System.Windows.Application.ShutdownCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
[02:26:20.0628132] > [Warning] Config version (-1) is not supported!
[02:26:20.0773833] > [Info]  Detected 5120x1440 Resolution.

You can adjust this inside the settings.
[02:26:20.0900610] > [Er
```
</details>

**Comment by Blightbuster** [comment]
It looks like you were forcing the application to close while it was still in the middle of something is this correct or does this happend on its own?

**Comment by ImJustMango** [comment]
Hi, I don't believe so, I tried running the program as an administrator and this happened.

My exact steps were as follows:

1. Make folder on desktop and extract to it
2. Run the application as administrator
3. Get error popup that closes in less than a second followed by popup that asks to create issue on github. 

Tarkov is open while launching if that matters.

**Comment by namrog84** [comment]
Same thing happens for me.

1. I launch RatScanner.exe
2. Rat scanner shows hard to read "Loading..." 
3. after a few seconds I get the above error and here I am

**Comment by Blightbuster** [comment]
Could you please try installing the following dependencies in the order provided and check after each one if RatScanner works.
If Rat Scanner is showing it's window but a its inside is only black, please be patient as it might need to download some stuff.

[WebView2 Runtime](https://go.microsoft.com/fwlink/p/?LinkId=2124703)

[Download ASP.NET Core 6.0 Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-aspnetcore-6.0.0-windows-hosting-bundle-installer)

[Download .NET 6.0 Desktop Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.0-windows-x64-installer)

**Comment by Blightbuster** [comment]
Fixed in [v3.2.1](https://github.com/RatScanner/RatScanner/releases/download/v3.2.1/RatScanner.zip).
Thanks for creating the issue.
